api/package.json

```diff
@@ -7,10 +7,19 @@
     "clean": "rm -rf build && mkdir build",
     "build-server": "babel -d ./build ./src -s",
     "build": "npm run clean && npm run build-server",
-    "start": "node ./build/index.js",
+    "start": "set NODE_ENV='PROD' && node ./build/index.js",
     "dev-start": "nodemon --exec babel-node ./src/index.js",
-    "test": "nyc mocha --require @babel/register ./src/test/ --exit || true",
-    "coverage": "nyc npm test && nyc report --reporter=text-lcov | coveralls"
+    "test": "set NODE_ENV='TEST' && npm run migrations && npm run seeders && nyc mocha --timeout=20000 --require @babel/register ./src/test/ --exit || true",
+    "singleTest": "set NODE_ENV='TEST' && nyc mocha --timeout=20000 --require @babel/register ./src/test/account.test.js --exit || true",
+    "coverage": "nyc npm test && nyc report --reporter=text-lcov | coveralls",
+    "migrateUser": "babel-node ./src/database/migrations/user createUserTable",
+    "migrateAccount": "babel-node ./src/database/migrations/account createAccountTable",
```

Comment:
why are you migrating the tables differently?

Reply:
I've merged it into one file now.